recipes/atlas-schema/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "atlas-schema" %}
+{% set version = "0.1.0" %}
+{% set python_min = "3.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/atlas_schema-{{ version }}.tar.gz
+  sha256: 50f8fd29e958f00ff5fe8b0f2235e281952e5391e893bb38e61c2c50751b7766
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  #skip: true  # [win]
```

Comment:
```suggestion
```

You can fully remove this line, as I remember now that a noarch build just builds on x86 Linux as the noarch bit makes other builds redundant. So there's no need to have this put back in the future.